actions_std.go

```diff
@@ -2160,6 +2160,21 @@ func scriptingActions() {
 			}
 		},
 	}
+	actions["openApp"] = actionDefinition{
+		identifier: "openapp",
```

Comment:
If the action key (e.g. `actions["key"]`) and the `identifier` are the same, the identifier field can be removed.